core/src/main/java/overflowdb/ReferenceManager.java

```diff
@@ -39,6 +37,15 @@
   public ReferenceManager(OdbStorage storage, NodesWriter nodesWriter) {
     this.storage = storage;
     this.nodesWriter = nodesWriter;
+    this.executorService = Executors.newSingleThreadExecutor(new NamedThreadFactory("overflowdb-reference-manager"));
+    this.shouldShutdown = false;
```

Comment:
why don't we want to shutdown the executorService if it's the default singlethreaded one? we used to, in the past...

Reply:
If it is used outside of overflowdb that would be undesired

Reply:
but i guess it should be the other way around in the constructors, Olof?

Reply:
Yes, that should've been `true` of course :+1: